addons/l10n_lb/data/template/account.tax.group-lb.csv

```diff
@@ -0,0 +1,4 @@
+"id","name","country_id","tax_receivable_account_id","tax_payable_account_id","preceding_subtotal","name@ar_001"
```

Comment:
You should remove the `preceding_subtotal` if you leave it empty in all your groups